Example-iOS/AppDelegate.swift

```diff
@@ -24,12 +24,12 @@ class AppDelegate: UIResponder, UIApplicationDelegate {
         guard let traitCollection = window?.traitCollection else {
             fatalError("There should be a traitCollection available before calling this method.")
         }
-        let titleStyle = StringStyle.style(
+        let titleStyle = StringStyle(parts:
```

Comment:
@dostrander if you can delete the parameter name and get it to compile on Swift 2.3 (change the setting in the Xcode project), I'd love to see it. I couldn't get it to work.